frontend/frontend/src/components/Profile.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+	<div>
+		<h2> {{this.$route.params.gameWon==true? "Congratulations": "Better Luck Next Time!"  }}</h2>
+		<h3> Time Stopped at: {{this.$route.params.timer }} </h3>
+		<h3> Match Count: {{ this.$route.params.matches }} </h3>
+
+    </div>
```

Comment:
I think this looks great! could we add a simple done button for now to route them back to the home page for completeness?